frappe/templates/includes/search_template.html

```diff
@@ -23,8 +23,8 @@ <h3 style="margin-bottom: 30px">{{ title }}</h3>
 	<form action='{{ route }}'>
 	<input name='q' class='form-control' type='text'
 		style='max-width: 400px; display: inline-block; margin-right: 10px;'
-		value='{{ frappe.form_dict.q or ''}}'
-		{% if not frappe.form_dict.q%}placeholder="{{ _("Search...") }}"{% endif %}>
+		value='{{ query or ''}}'
```

Comment:
is `query` bleached?

Reply:
Yes
https://github.com/frappe/frappe/blob/21f5736b15598eec9d087b9912fa1fb3aa4acbcb/frappe/www/search.py#L12-L14